base/deprecated.jl

```diff
@@ -1495,11 +1495,11 @@ function replace(s::AbstractString, pat, f, n::Integer)
 end
 
 # PR #22475
-@deprecate ntuple{N}(f, ::Type{Val{N}}) ntuple(f, Val(N))
-@deprecate fill_to_length{N}(t, val, ::Type{Val{N}}) fill_to_length(t, val, Val(N)) false
-@deprecate literal_pow{N}(a, b, ::Type{Val{N}}) literal_pow(a, b, Val(N)) false
-@eval IteratorsMD @deprecate split{n}(t, V::Type{Val{n}}) split(t, Val(n)) false
-@deprecate sqrtm{T,realmatrix}(A::UpperTriangular{T},::Type{Val{realmatrix}}) sqrtm(A, Val(realmatrix))
+@deprecate ntuple(f, ::Type{Val{N}}) where N  ntuple(f, Val(N))
```

Comment:
still going to show up in `@which`, and it's newly added so going to be around for a bit